backend/src/main/java/capstone/myfinancemanager/manager/model/RandomUUIDGenerator.java

```diff
@@ -0,0 +1,13 @@
+package capstone.myfinancemanager.manager.model;
+
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+@Service
+public class RandomUUIDGenerator {
+
+    public String getRandomId() {
```

Comment:
Warum die Klasse einzeln und nicht in Transaction.java

Reply:
Ich habe das für die Test ausgegelaggert. So kann ich die Klassen mocken und das verhalten festlegen.